spec/index.html

```diff
@@ -9765,20 +9787,38 @@ <h5>GroupConcat</h5>
               SEPARATOR.</p>
             <div class="defn">
               <p><b>Definition: <span id="defn_aggGroupConcat">GroupConcat</span></b></p>
-              <pre class="code nohighlight">literal GroupConcat(sequence S, function scalarvals)</pre>
-              <p>If the scalarvals argument is absent from GROUP_CONCAT, then scalarvals is taken to
-                be the empty function.</p>
-              <p>|separator| = scalarvals("separator") if scalarvals is defined for the argument "separator"</p>
-              <p>|separator| = the "space" character, unicode codepoint U+0020, if scalarvals is undefined for the argument "separator"</p>
-              <p>L = Flatten(S)</p>
-              <p>GroupConcat(S, scalarvals) = GroupConcat(L, |separator|)</p>
-              <p>GroupConcat(L, sep) = "" if <a href="#defn_Card">Card</a>(L) = 0</p>
-              <p>GroupConcat(L, sep) = CONCAT("", L<sub>1</sub>) if
-                <a href="#defn_Card">Card</a>(L) = 1</p>
-              <p>GroupConcat(L, sep) = CONCAT(L<sub>1</sub>, sep, GroupConcat(L<sub>2..n</sub>,
-                sep)) if <a href="#defn_Card">Card</a>(L) &gt; 1</p>
-            </div>
-            <p>For example, GroupConcat([("a"), ("b"), ("c")], {"separator" → "."}) = "a.b.c".</p>
+              <pre class="code nohighlight">literal <var>GroupConcat</var>(sequence <var>S</var>, function <var>scalarvals</var>)</pre>
+              <p>If the <var>scalarvals</var> argument is absent from GROUP_CONCAT,
+                then <var>scalarvals</var> is taken to be the empty function.</p>
+              <p>Let <var>sep</var> be a string that is defined as follows.</p>
+              <ul>
+                <li>If <var>scalarvals</var> is defined for the argument "separator",
+                  then <var>sep</var> = <var>scalarvals</var>("separator").</li>
+                <li>If <var>scalarvals</var> is undefined for the argument "separator",
+                  then <var>sep</var> is the "space" character (i.e., unicode codepoint U+0020).</li>
+              </ul>
+              <p><var>GroupConcat</var>(<var>S</var>, <var>scalarvals</var>) =
+                <var>GroupConcat'</var>(<var>L</var>, <var>sep</var>),</p>
+              <p>where <var>L</var> = Flatten(<var>S</var>) and
+                <var>GroupConcat'</var>(<var>L</var>, <var>sep</var>)
+                is defined recursively as follows.</p>
+              <ul>
+                <li>If <a href="#defn_Card">Card</a>(<var>L</var>) = 0, then
+                  <var>GroupConcat'</var>(<var>L</var>, <var>sep</var>) = "".</li>
+                <li>If <a href="#defn_Card">Card</a>(<var>L</var>) = 1, then
+                  <var>GroupConcat'</var>(<var>L</var>, <var>sep</var>) =
+                  CONCAT("", <var>L</var><sub>1</sub>).</li>
```

Comment:
```suggestion
                  <code>CONCAT</code>("", <var>L</var><sub>1</sub>).</li>
```

Reply:
It is good to put the `<code>` on GROUP_CONCAT as per first suggestion where it is in running text.

Here, inside already formatted text for a definition, it's the abstract "concat" like the GroupConcat being defined. The reSpec autolinking should work here. 
I find the `<code>` at this point unhelpful. The style does not, and should not, apply to other names.

Reply:
> [@afs] Here, inside already formatted text for a definition, it's the abstract "concat" like the GroupConcat being defined. The reSpec autolinking should work here.

Autolinking does not work because there is no `<dfn>CONCAT</dfn> in the document. However, I have now explicitly linked the function name CONCAT in this definition of GroupConcat to the section that defines the CONCAT function. This change is implemented in commit https://github.com/w3c/sparql-query/pull/124/commits/745017a9d1c072ffea2ea39c43581fd54b051a0f